tools/depth-quality/readme.md

```diff
@@ -29,10 +29,12 @@ You should be able to easily get and interpret several of the depth quality metr
 ## Metrics elaborated
 ![](./res/Zi_ZPi.png)
 
-### Depth Error Average & STD
+### Depth Error Average
 _Dist<sub>i</sub>_ - Distance from pixel coordinates to the Plane Fit (mm)  
-![](./res/avg.gif)  
-![](./res/std.gif)
+![](./res/avg.gif)
+### Z Accuracy
+_GT_ - Ground truth distance to the wall (mm)  
+![](./res/accuracy.gif)
```

Comment:
Please modify the formula for Z Accuracy so if won't use the Dist_i as it is plane-fit related
I think it is also worth mention that as the angle (skew) is grows then the Z accuracy will be less and less accurate, especially if using large ROI
And to get the actual Z accuracy this values shall not exceed some threshold value.